Content.Server/Forensics/Components/ForensicScannerComponent.cs

```diff
@@ -0,0 +1,20 @@
+using System.Threading;
+using Content.Server.UserInterface;
+using Content.Shared.Forensics;
+using Robust.Server.GameObjects;
+
+namespace Content.Server.Forensics
+{
+    [RegisterComponent]
+    public sealed class ForensicScannerComponent : SharedForensicScannerComponent
+    {
+        public BoundUserInterface? UserInterface => Owner.GetUIOrNull(ForensicScannerUiKey.Key);
```

Comment:
don't use `Owner.GetUIOrNull`, use the `UserInterfaceSystem` methods for these (and don't put it on a property as well). the reason is that the `GetUIOrNull` overload hides some pretty slow method calls and is intended to be removed at some point